app/src/androidTest/java/com/github/geohunt/app/TestHelpers.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.geohunt.app
+
+import com.google.android.gms.tasks.Task
+import kotlinx.coroutines.CompletableDeferred
+import java.util.concurrent.CompletableFuture
+import kotlin.coroutines.cancellation.CancellationException
+

```

Comment:
The code is easy to understand, but it would be helpful to have a Javadoc explaining what the functions do and how they should be used.